api/data_explorer/__main__.py

```diff
@@ -224,13 +225,30 @@ def _get_es_facets():
             facets[ui_facet_name] = TermsFacet(field=field_name + '.keyword')
         else:
             # Assume numeric type.
-            # TODO: Handle other types.
-            # TODO: Automatically figure out bucket intervals. Unfortunately
-            # Elasticsearch won't do this for us
-            # (https://github.com/elastic/elasticsearch/issues/9572). Make the
-            # ranges easy to read (10-19,20-29 instead of 10-17,18-25).
+            # Creating this facet is a two-step process.
```

Comment:
Since this function is getting pretty long and has substeps and sub-documentation, split out to helper methods `_get_max_value` and `_get_bucket_size`?

Reply:
Done